library/src/main/java/com/pokegoapi/util/PokeNames.java

```diff
@@ -91,4 +91,17 @@ public static int getPokedexFromName(String pokeName)
 			throws UnsupportedEncodingException, MissingResourceException {
 		return getPokedexFromName(pokeName, Locale.ENGLISH);
 	}
+
+	/**
+	 * Returns the Pokemon index from the Pokemon name list which matches in PokemonId enums.
+	 *
+	 * @param protoPokeName the Pokemon proto enum name
+	 * @return pokedex
+	 * @throws UnsupportedEncodingException if the named charset is not supported
+	 * @throws MissingResourceException if can not find a matched Pokemon name for the given pokedex
+	 */
+	public static int getPokedexFromProtoName(String protoPokeName)
+			throws UnsupportedEncodingException, MissingResourceException {
+		return getPokedexFromName(protoPokeName.replace("_FEMALE", "\u2640").replace("_MALE", "\u2642").replace("_", ". "), Locale.ENGLISH);
```

Comment:
Oops, you right I forgot about this.
I was stuck on CatchablePokemon.getPokemonId(), but did not realize there is a getNumber() method available after that...
Gonna remove this right now